### ISSUE TITLE
 Error: updating API Gateway Stage failed: BadRequestException: CloudWatch Logs role ARN must be set in account settings to enable logging

### ISSUE THREAD

**Issue by eefret** [StratusGrid/terraform-aws-transfer-server-custom-idp#7]
## Expected Behavior
API Gateway Stage created succesfully

## Actual Behavior
Error: updating API Gateway Stage failed: BadRequestException: CloudWatch Logs role ARN must be set in account settings to enable logging

## Steps to Reproduce the Problem

  1. Running the TF apply

## Specifications

  - Version: 1.0.2
  - Platform: macOS | terraform 1.0.7
  - Subsystem:
```
Terraform v1.0.7
on darwin_amd64
+ provider registry.terraform.io/hashicorp/archive v2.2.0
+ provider registry.terraform.io/hashicorp/aws v3.74.3
+ provider registry.terraform.io/hashicorp/azuread v2.18.0
+ provider registry.terraform.io/hashicorp/azurerm v2.98.0
+ provider registry.terraform.io/hashicorp/cloudinit v2.2.0
+ provider registry.terraform.io/hashicorp/null v3.1.0
+ provider registry.terraform.io/hashicorp/tls v3.1.0
```
![Screen Shot 2022-04-14 at 12 20 06](https://user-images.githubusercontent.com/863005/163431589-6a9731d2-ab8e-49cc-bdb2-6d18f95c0a58.png)

**Comment by eefret** [comment]
Workaround is setting `metrics_enabled = false` on apigateway.tf line 111 

```
resource "aws_api_gateway_method_settings" "get" {
  rest_api_id = aws_api_gateway_rest_api.sftp.id
  stage_name  = aws_api_gateway_stage.prod.stage_name
  method_path = "*/*"

  settings {
    metrics_enabled = false
    logging_level   = "ERROR"
  }
}
```

**Comment by eefret** [comment]
will probably submit a PR later this weekend great job on this OP

**Comment by angellopez-sg** [comment]
I've tried to reproduce the issue without success :S

**Comment by chrischildresssg** [comment]
@angellopez-sg the issue is that the logging role for API Gateway must be assigned account-wide; it cannot be assigned per API Gateway.  Honestly, this role should be created outside of the module and passed in as a parameter/variable.

**Comment by chrischildresssg** [comment]
It is my understanding that this issue has been resolved as of v1.2.0 (PR #18 ). Closed.
